core/src/main/java/contacts/core/Fields.kt

```diff
@@ -180,7 +180,8 @@ object Fields : AbstractDataFieldSet<AbstractDataField>() {
     @JvmField
     val Phone = PhoneFields()
 
-    internal val Photo = PhotoFields()
+    @JvmField
+    val Photo = PhotoFields()
```

Comment:
Exposed Photo field and entity to consumers because references to it are now a part of a public interface.